src/BlazorWebView/tests/MauiDeviceTests/WebViewHelpers.Windows.cs

```diff
@@ -45,13 +34,14 @@ public static async Task WaitForWebViewReady(WebView2 wv2)
 				// It's possible that the DOMContentLoaded event won't fire because it's already loaded by the time we got here. To check
 				// for that, we inspect an arbitrary custom HTML element attribute to see if we can find it. If we can find it, then surely
 				// the DOM content is loaded, so we can continue with the test.
-				var testHtmlLoadedAttributeValue = await wv2.CoreWebView2.ExecuteScriptAsync("(document.head.attributes['testhtmlloaded']?.value === 'true')");
 
-				if (testHtmlLoadedAttributeValue != "true")
+				await Retry(async () =>
```

Comment:
This code here never had a retry (despite the exception message saying it did!). And this is where the test failure that motivated this change occured - so hopefully adding retries mitigates that particular flakiness.